resource/js/app.js

```diff
@@ -35,3 +39,8 @@ Object.keys(componentMappings).forEach((key) => {
     ReactDOM.render(componentMappings[key], elem);
   }
 });
+
+// うわーもうー
+$('a[data-toggle="tab"][href="#revision-history"]').on('show.bs.tab', function() {
```

Comment:
We can use `document.querySelector('a[data-toggle="tab"][href="#revision-history"]')`.
But I did not test this event using it.